src/uniprotkb/components/protein-data-views/SubcellularLocationWithVizView.tsx

```diff
@@ -0,0 +1,240 @@
+const getStylesContaining = (...selectors: string[]) =>
+  Array.from(document.styleSheets)
+    .flatMap((stylesheet) =>
+      Array.from(stylesheet.rules).map(
+        (rule) =>
+          selectors.some((selector) => rule.cssText.includes(selector)) &&
+          rule.cssText
+      )
+    )
+    .filter(Boolean) as string[];
+
+const SubcellularLocationWithVizView: FC<
+  {
+    comments?: SubcellularLocationComment[];
+  } & Pick<OrganismData, 'taxonId' | 'lineage'>
+> = ({ comments, taxonId, lineage }) => {
+  /*
+    The logic implemented here to get our data into @swissprot/swissbiopics-visualizer has been lifted
+    from the source code at http://sp.sib.swiss/scripts/uniprot_entry-bundle.js with some modifications
+  */
+  const ref = useRef<HTMLDivElement>(null);
+  const [ceLoaded, setCELoaded] = useSafeState(false);
+
+  useEffect(() => {
+    import('@swissprot/swissbiopics-visualizer').then(
+      () => setCELoaded(true),
+      // eslint-disable-next-line no-console
+      (error) => console.error(error)
+    );
+  }, [setCELoaded]);
+
+  // NOTE: injecting HTML because of the way the web component is implemented.
+  //       See here for details: https://stackoverflow.com/questions/43836886/failed-to-construct-customelement-error-when-javascript-file-is-placed-in-head
+  //       After discussion with author of @swissprot/swissbiopics-visualizer this will most likely not change.
+  // TODO: add additional GO template which will also require GO term data
+  // TODO: handle this case as seen in the source code 'svg .membranes .membrane.subcell_present' eg A1L3X0 doesn't work
+  // TODO: clicking a UniProtKBEvidenceTag doesn't open the content
+  useEffect(() => {
+    if (lineage && taxonId && comments && !isVirus(lineage)) {
+      const sls = comments
+        ?.flatMap(({ subcellularLocations }) =>
+          subcellularLocations?.map(({ location }) =>
+            getSubcellularLocationId(location.id)
+          )
+        )
+        .filter(Boolean)
+        .join(',');
+
+      const tabsHeaderHeight = document.querySelector('.tabs__header')
+        ?.clientHeight;
+      const pictureTop = tabsHeaderHeight
+        ? `${tabsHeaderHeight + 5}px`
+        : '4rem';
+
+      ref.current?.insertAdjacentHTML(
+        'afterbegin',
+        `
+      <template id="sibSwissBioPicsStyle">
+        <style>
+          #swissBioPicsSlData {
+            grid-area: terms;
+          }
+          .lookedAt {
+            background-color: #abc7d6 !important;          
+            stroke: black !important;
+            fill: #abc7d6 !important;
+            fill-opacity: 1 !important;
+          }
+          #swissbiopic > svg {
+            position: sticky;
+            top: ${pictureTop};
+          }
+          .subcell_name {
+            display: none;
+          }
+          .subcell_description {
+            display: none;
+          }
+          .inpicture:before{
+            content: '🔎';
```

Comment:
```suggestion
            content: '🔎 ';
```
Just to have a bit of breathing room after the icon

Reply:
I had a right margin set for the icon but is a space character preferable?

Reply:
Ah, this wasn't visible somehow when I tested it, that's why I added a space. Will have another look on Monday